doc/classes/Image.xml

```diff
@@ -330,6 +330,7 @@
 			</argument>
 			<description>
 				Loads an image from file [code]path[/code].
+				Returns 0 ([code]OK[/code]) if the load was successful, otherwise an error code from [@GlobalScope].
```

Comment:
You don't need to mention `0`, only `Returns [code]OK[/code]`. People should check against `OK`, not magic numbers.

Reply:
But basically the whole sentence doesn't say much more than the
```
<return type="int" enum="Error">
</return>
```
in my opinion. If we add it here, we would need to add it to all methods that return `Error` and where we don't already document explicitly what kind of error codes can come out.

Reply:
Yeah now that enums are properly documented by the doc writer this could probably be simplified.